meetings/maintainers/2024/maintainers-2024-06-10.md

```diff
@@ -0,0 +1,65 @@
+# GUAC Maintainers Meeting: 2024-06-10
+
+Recording: https://youtu.be/qp1EwXQXXcM
+
+## Attendees
+
+| Name | Email | Affiliation | Pronouns
+| ---- | ----- | ----------- | --------
+| Marco Rizzi | mrizzi@redhat.com | Red Hat | he/him
+| Jeff Mendoza | jlm@jlm.name | Kusari | he/him
+| Ben Cotton | ben@kusari.dev | Kusari | he/him
+| Mike Lieberman | mike@kusari.dev | Kusari | he/him
+| Marco Deicas | mdeicas@google.com | Google | he/him
+| Parth Patel | parth@kusari.dev | Kusari | he/him
+| Mihai Maruseac | mihaimaruseac@google.com | Google | he/him
+| Narsa Chelluri | narsa@kusari.dev | Kusari | he/him
+| Brandon Lum | lumb@google.com | Google | he/him
+| Reden Martinez | rmartinez@linuxfoundation.org | Linux Foundation | he/him
+| Karla Obermeier
```

Comment:
This seems to be an incomplete row

Reply:
Karla only put her name in the Google Doc, so there's nothing else to add. The table renders correctly, so there's nothing to do here unless you'd like me to add the `|` for completeness.

Reply:
Oh, somehow GitHub was taking too long for me to render and I assumed it's because of that. Might have been an issue on their side, so this is good to go. Thnak you!